gson/src/test/java/com/google/gson/stream/JsonWriterTest.java

```diff
@@ -762,6 +850,7 @@ public void testStrictWriterDoesNotPermitMultipleTopLevelValues() throws IOExcep
       writer.beginArray();
       fail();
     } catch (IllegalStateException expected) {
+      assertThat(expected).hasMessageThat().isEqualTo("JSON must have only one top-level value.");
```

Comment:
This test method might be redundant because `testMultipleTopLevelValues` seems to test the same.

(And maybe `testLenientWriterPermitsMultipleTopLevelValues` should then be moved up to the other top-level test methods.)